src/connector.js

```diff
@@ -116,6 +117,18 @@ function initSearchUI() {
 	paramsDetect.isAdvancedSearch = !!document.getElementById( 'advseacon1' ) || winPath.endsWith( '/advanced-search.html' ) || winPath.endsWith( '/recherche-avancee.html' );
 	paramsDetect.enableHistoryPush = !paramsDetect.isAdvancedSearch;
 
+	// Auto detect relative path from originLevel3
+	if ( paramsOverride.originLevel3 ) {
+		try {
```

Comment:
Can you please explain the rationale for having a try catch? I simply don't think this block deserves a try catch.

Reply:
After switching my mindset to the newer approach I am proposing, the if is not needed at all anymore. The approach proposed will set the originLevel3Relative after the `originLevel3` has been either overridden through the data attribute or through URL parameter, or even just to set it based on its default value (which is defined at line 38).